Sources/ObjCSupport/LOKLabelLayout.swift

```diff
@@ -13,11 +13,13 @@ import UIKit
     @objc public let string: String?
     @objc public let lineHeight: CGFloat
     @objc public let font: UIFont
+    @objc public let lineBreakMode: NSLineBreakMode?
     @objc public let numberOfLines: Int
     @objc public let alignment: LOKAlignment
     @objc public let viewClass: UILabel.Type
     @objc public let configure: ((UILabel) -> Void)?
 
+    // TODO: Remove this init once all consumers are switch to `init(attributedString:font:lineBreakMode:lineHeight:numberOfLines:alignment:flexibility:viewReuseId:viewClass:configure:)`
```

Comment:
I think we should just add the new parameter, since it's optional.

Reply:
- We have added a new parameter in LOKLabelLayout.swift but it will be a breaking change because every Objective-C caller will have to pass the new parameter else it won't build. Also we made it non-optional parameter as optional enum won't be accessible in Objective-C. 
Let us know if this should be fine.